widget/button.go

```diff
@@ -21,7 +21,8 @@ type ButtonIconPlacement int
 // ButtonImportance represents how prominent the button should appear
 //
 // Since: 1.4
-type ButtonImportance int
+// Deprecated: Use widget.Importance instead
+type ButtonImportance Importance
```

Comment:
Your change is currently backwards-incompatible as any code looking like this won't compile:
```go
var importance widget.ButtonImportance = widget.MediumImportance
```
You can fix it by using an equal sign like mentioned in https://github.com/fyne-io/fyne/pull/4047#issuecomment-1631252683:
```suggestion
type ButtonImportance = Importance
```